commands/store.go

```diff
@@ -23,12 +24,15 @@ var (
 )
 
 const (
-	defaultStore      = "https://cdn.rawgit.com/openfaas/store/master/store.json"
+	defaultStore      = "https://cdn.rawgit.com/openfaas/store/master/functions.json"
 	maxDescriptionLen = 40
 )
 
+var platformValue string
+
 func init() {
 	storeCmd.PersistentFlags().StringVarP(&storeAddress, "url", "u", defaultStore, "Alternative Store URL starting with http(s)://")
+	storeCmd.PersistentFlags().StringVarP(&platformValue, "platform", "p", "", "Target platform for store")
```

Comment:
Should we put the default platform `defaultStore` here, instead of `""`, maybe work with `platformValue` directly? This way we can remove this package `platform/platform.go`

Reply:
Adding a default value would make it more obvious when using the help commands as well

It could be added as a variable as well. With the current implementation, it would look like this:

`..., "p", platform.Platform, "Target platform...`

That would use the build parameter input and show as the default value in the help text:

```
Flags:
  -h, --help              help for store
  -p, --platform string   Target platform for store (default "x86_64")

```

Reply:
@viveksyngh did you do this?

Reply:
`platform` package has been removed and we are using `Platform` variable in `command` package to inject to value at build time and same value is being used as default value for the flag.